src/app/ui.js

```diff
@@ -49,6 +57,40 @@ class UI {
     const newTask = document.querySelector('.task-input');
     newTask.value = '';
   }
+
+  static dragUI(e) {
+    e.target.classList.add('drag');
+  }
+
+  static stopDrag(e) {
+    e.target.classList.remove('drag');
+  }
+
+  static dragOver() {
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
- When i try to move a task at the top of the list i can't do it